### PR TITLE
Add observability bundle

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR:

- adds the observability bundle

### Testing

Description on how `observability bundle` can be tested.

- [x] fresh install works

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Review by vxav — COMMENTED** [review]
@erkanerol should we wait till you have a working Guppy before approving this?

**Review by mcbenjemaa — APPROVED** [review]

File: helm/default-apps-cloud-director/values.schema.json [hunk 1]
```diff
@@ -109,10 +400,5 @@
                 }
             }
         }
-    },
-    "required": [
```
Comment (erkanerol) [inline helm/default-apps-cloud-director/values.schema.json:113]: I think this `required` section is deleted accidentally.

File: helm/default-apps-cloud-director/values.schema.json [hunk 2]
```diff
@@ -3,66 +3,357 @@
...
         "clusterName": {
-            "minLength": 1,
             "type": "string"
         },
         "managementCluster": {
-            "minLength": 1,
             "type": "string"
         },
         "organization": {
-            "minLength": 1,
```
Comment (erkanerol) [inline helm/default-apps-cloud-director/values.schema.json:65]: Should stay

File: helm/default-apps-cloud-director/values.schema.json [hunk 3]
```diff
@@ -3,66 +3,357 @@
     "type": "object",
     "properties": {
         "apps": {
-            "additionalProperties": {
```
Comment (erkanerol) [inline helm/default-apps-cloud-director/values.schema.json:6]: why do we change `additionalProperties`?

Comment (QuentinBisson) [reply helm/default-apps-cloud-director/values.schema.json:6]: I ran helm schema-gen so it removed all of those. Do you manage the schéma by hand?

Comment (erkanerol) [reply helm/default-apps-cloud-director/values.schema.json:6]: yeah :(

**Comment by QuentinBisson** [comment]
@erkanerol can you take another look?

**Comment by QuentinBisson** [comment]
Apart from the wrong organization set on the cluster, it was tested successfully on Guppy :)

**Pull request merged**